packages/sdk/wallets/wallet-ledger/src/ledger-signer.ts

```diff
@@ -59,13 +59,14 @@ export class LedgerSigner implements Signer {
       }
       signature.v = addToV.toString(10)
       return {
-        v: signature.v,
+        v: parseInt(signature.v, 10), // TODO check if this should be a 10 and not a 16
```

Comment:
above in one spot its parsed as 16 but then line 60 shows signature.v set to a toString(10)

Reply:
10 seems right since above signature.v is set to addToV as base to